### ISSUE TITLE
ExtJS TypeError when opening Grid Options for Data Object Folder

### ISSUE THREAD

**Issue by mattsches** [pimcore/pimcore#8408]
# Bug Report
I'm on Pimcore 6.8.10 (latest release as of now) and have created a folder structure with DataObjects in subfolders. After I select a folder from the tree on the left, I get a list of my DataObjects in that folder. When I open the "Grid Options", the backend GUI breaks. I can sometimes close the "popup" window for "Grid Options", but nothing else will load after that.

In the browser's (Vivaldi, Chrome-based; in Firefox the error message is slightly different) JS console, I get this error
```
ext-all.js?_dc=162523ef6c0f3735c26c66c6627c9725ba337dc2:22 Uncaught TypeError: Cannot read property 'isModel' of null
    at g.createNode (ext-all.js?_dc=162523ef6c0f3735c26c66c6627c9725ba337dc2:22)
    at g.appendChild (ext-all.js?_dc=162523ef6c0f3735c26c66c6627c9725ba337dc2:22)
    at klass.initLayoutFields (script-proxy?scripts=minified_javascript_core_c0cdfa233fdf7ea64ade50e3e3aad1d9.js&_dc=162523ef6c0f3735c26c66c6627c9725ba337dc2:37211)
    at Object.callback (ext-all.js?_dc=162523ef6c0f3735c26c66c6627c9725ba337dc2:22)
    at E.onComplete (ext-all.js?_dc=162523ef6c0f3735c26c66c6627c9725ba337dc2:22)
    at E.onStateChange (ext-all.js?_dc=162523ef6c0f3735c26c66c6627c9725ba337dc2:22)
    at XMLHttpRequest.t (ext-all.js?_dc=162523ef6c0f3735c26c66c6627c9725ba337dc2:22)
```
### Expected behavior
Editing the Grid Options of a Folder in the backend should be possible and not break the whole ExtJS based backend GUI.

### Actual behavior
The backend becomes unresponsive for the most part, other views do not load anymore, Grid Options cannot be changed.

### Steps to reproduce

https://user-images.githubusercontent.com/68414/110776432-59344080-8260-11eb-83c8-49bbeed8291a.mp4

**Comment by dvesh3** [comment]
Cannot Reproduce on 6.8.10. @mattsches could you please share more insights or exact steps to reproduce? thanks!

**Comment by mattsches** [comment]
Seems the team figured it out:

In our class definition (Settings > Data Objects > Classes), we had added a Text Input Data Component directly under General Settings. Which isn't even possible by right-clicking on General Settings (only Layout Components can be added) but can be achieved by copy & paste.

So we had a structure like this:
![grafik](https://user-images.githubusercontent.com/68414/110944771-2d3abd00-833d-11eb-9e7b-c56b12b63afd.png)
and this somehow broke something in ExtJs.

After moving `myTextInput` into `myPanel`, everything works fine again 😀

Although I want to mention that I've just created a **new** class definition with the same "wrong" structure, and the error did not reappear. So all is good for now, I guess!
